package/CHANGELOG

```diff
@@ -13,6 +13,15 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+07/09/18 arm61
+
+  * 0.18.1
+
+Fixes
+
+  * Introduced compatibility for packmol (and hopefully generally) for pbd files with
```

Comment:
insert this inside the existing chunk below, also add yourself to the AUTHORS file as it's your first contribution